Sources/SentrySwiftUI/SentryTracerView.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+import Sentry
+import SwiftUI
+#if SWIFT_PACKAGE
+import SentryInternal
+#endif
+
+/// A control to measure the performance of your views and send the result as a transaction to Sentry.io.
+///
+/// You create a transaction by wrapping your views with this.
+/// Nested `SentryTracerView` will create child spans in the transaction.
+///
+///     SentryTracerView {
+///         VStack {
+///             // The part of your content you want to measure
+///         }
+///     }
+///
+/// By default, the transaction name will be the first root view, in the case above `VStack`.
+/// You can give your transaction a custom name by providing the name parameter.
+///
+///     sentryTrace("My Awesome Screen") {
+///         VStack {
+///             // The part of your content you want to measure
+///         }
+///     }
+///
+/// Alternatively you can use the view extension:
+///
+///     VStack {
+///         //The part of your content you want to measure
+///     }.sentryTransaction("My Awesome Screen")
+///
+///
+@available(iOS 13, macOS 10.15, tvOS 13, watchOS 6.0, *)
+public struct SentryTracerView<Content: View>: View {
```

Comment:
`m`: I think it should be
```suggestion
public struct SentryTraceView<Content: View>: View {
```